src/Lunr/Ticks/PHPMailer/PHPMailer.php

```diff
@@ -0,0 +1,281 @@
+     *
+     * @return void
+     */
+    protected function afterSending(bool $isSent, array $to, array $cc, array $bcc, string $subject, string $body, string $from, array $extra): void
+    {
+        if ($this->analyticsDetailLevel === AnalyticsDetailLevel::None)
+        {
+            return;
+        }
+
+        $endTimestamp = microtime(TRUE);
+
+        $fields = [
+            'startTimestamp' => $this->startTimestamp,
+            'endTimestamp'   => $endTimestamp,
+            'executionTime'  => (float) bcsub((string) $endTimestamp, (string) $this->startTimestamp, 4),
+            'url'            => $this->Host, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+            'ip'             => gethostbyname($this->Host), // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+            'traceID'        => $this->tracingController->getTraceId(),
+            'spanID'         => $this->tracingController->getSpanId(),
+            'parentSpanID'   => $this->tracingController->getParentSpanId(),
+        ];
+
+        if ($this->analyticsDetailLevel->atleast(AnalyticsDetailLevel::Detailed))
+        {
+            $options = [
+                'from' => $from,
+                'to'   => $to,
+                'cc'   => $cc,
+                'bcc'  => $bcc,
+            ];
+
+            // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+            if ($this->Mailer === 'smtp')
+            {
+                $options += [
+                    'SMTPHost'      => $this->Host, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPPort'      => $this->Port, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPHelo'      => $this->Helo, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPSecure'    => $this->SMTPSecure,
+                    'SMTPAutoTLS'   => $this->SMTPAutoTLS,
+                    'SMTPAuth'      => $this->SMTPAuth,
+                    'SMTPUsername'  => $this->Username, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPPassword'  => $this->Password, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPKeepAlive' => $this->SMTPKeepAlive,
+                    'SMTPAuthType'  => $this->AuthType, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                    'SMTPTimeout'   => $this->Timeout, // phpcs:ignore Lunr.NamingConventions.CamelCapsVariableName
+                ];
+
+                $options += $this->SMTPOptions;
+            }
+
+            $options = json_encode($options + $extra);
+
+            $fields['requestHeaders'] = $this->prepareLogData($this->MIMEHeader);
+            $fields['data']           = $this->prepareLogData($this->MIMEBody);
+            $fields['options']        = $this->prepareLogData(is_bool($options) ? '' : $options);
```

Comment:
Don't truncate `options` and `requestHeaders` as those might contain vital information (like, To, CC, BCC, etc). We don't do that for `Requests` either. Leave the truncating for the potentially large `data` and `responseBody` (though the latter is likely irrelevant here)